setup.cfg

```diff
@@ -15,8 +15,8 @@ classifier =
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: 3 :: Only
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.11
```

Comment:
Technically we should be listing all applicable Python versions here, not just the target SLE deployment version, so we should really have entries for 3.8 through 3.11, and Python 3.6 and 3.7 entries are still valid, if outdated.